doc/iterations/2024.5/shapes/create_representation_using_content_of_another_representation.adoc

```diff
@@ -0,0 +1,92 @@
+= Create a representation using the content of another representation as input
+
+== Context
+
+Representation event processors are isolated on purpose, but now we want to be able to synchronize the state of a representation event processor with another one.
```

Comment:
It's a more limited use case that what is described here, but note that the _Representations_ view (in the right panel) supports displaying the structure/contents of a _Portal_ representation (see https://github.com/eclipse-sirius/sirius-web/pull/2897).